mlrun/common/schemas/api_gateway.py

```diff
@@ -77,7 +77,7 @@ class APIGatewaySpec(_APIGatewayBaseModel):
     name: str
     description: Optional[str]
     path: Optional[str] = "/"
-    authenticationMode: Optional[APIGatewayAuthenticationMode] = (
+    authentication_mode: Optional[APIGatewayAuthenticationMode] = (
```

Comment:
We need it like this because we convert it to json [here](https://github.com/mlrun/mlrun/blob/3c9f80ed300620305399101b59f0620fd3d6c363/server/api/utils/clients/async_nuclio.py#L112)  and then it's used in nuclio - https://github.com/nuclio/nuclio/blob/672b8e36f9edd6e42b4685ec1d27cabae3c5f045/pkg/platform/types.go#L476

Reply:
Thanks, fixed